python/lsst/obs/base/_fitsRawFormatterBase.py

```diff
@@ -41,6 +41,10 @@ class FitsRawFormatterBase(FitsExposureFormatter, metaclass=ABCMeta):
     FITS files.
     """
 
+    # This has to be explicit until we fix camera geometry in RFC-605
+    wcsFlipX = False
+    """Control whether the WCS is flipped in the X-direction"""
```

Comment:
[Style guide says](https://developer.lsst.io/python/numpydoc.html#documenting-constants-and-class-attributes) to explicitly say this is `bool`.